lib/utils.ts

```diff
@@ -77,3 +83,7 @@ async function processContent(content: string) {
 function getSlugFromFileName(fileName: string) {
   return fileName.replace(/\.md$/, '').substring(11)
 }
+
+function finishedYear({ finishedOn }) {
+  return finishedOn.slice(0, 4)
+}
```

Comment:
Could this be moved to a variable function?

Reply:
https://github.com/kplattret/kevinplattret.com/pull/35/commits/d7fa1c286ee85680039a31e51307e97b6548c1a2